### PR TITLE
Issue29

### PR DESCRIPTION
**Pull request by knabben**

### CODE REVIEW

**Comment by knabben** [comment]
- Spanish menu file must be translated
- Portuguese must be corrected

File: fixtures/initial_data.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
         "model": "sites.site", 
         "fields": {
             "domain": "dq276.o1.gondor.io", 
-            "name": "SciPy 2014"
+            "name": "SciPy 2016"
```
Comment (rgaiacs) [inline fixtures/initial_data.json]: SciPy -> SciPyLA

File: fixtures/initial_data.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
         "model": "sites.site", 
         "fields": {
             "domain": "localhost", 
-            "name": "SciPy 2014 [local]"
+            "name": "SciPy 2016 [local]"
```
Comment (rgaiacs) [inline fixtures/initial_data.json]: SciPy -> SciPyLA

File: fixtures/sitetree_pt.json [hunk 3]
```diff
@@ -0,0 +1,418 @@
...
+        "fields": {
+            "parent": 102, 
+            "access_restricted": false, 
+            "access_permissions": [], 
+            "description": "", 
+            "insitetree": false, 
+            "hint": "", 
+            "url": "floorplans", 
+            "inbreadcrumbs": false, 
+            "title": "Venue Floor Maps",
```
Comment (rgaiacs) [inline fixtures/sitetree_pt.json]: "Venue Floor Maps" -> "Mapa do local do evento"

**Comment by rgaiacs** [comment]
@knabben Thanks. This pull request need just a few corrections and should be read to merge. :sparkles:

**Pull request merged**